lib/rpc/src/lib.rs

```diff
@@ -0,0 +1,26 @@
+// Copyright © 2021 University of Colorado. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+#![no_std]
+
+#[macro_use]
+extern crate abomonation;
+extern crate alloc;
```

Comment:
I think you don't need the extern crates anymore with rust 2018, e.g. should be fine to just delete these lines.

for abomonation, you might have to explicitly include macros in the right files